lib/Data/Chronicle/Writer.pm

```diff
@@ -8,14 +8,6 @@ use warnings;
 
 Data::Chronicle::Writer - Provides writing to an efficient data storage for volatile and time-based data
 
-=head1 VERSION
-
-Version 0.05
-
-=cut
-
-our $VERSION = '0.05';
```

Comment:
Might be worth inheriting $VERSION from Data::Chronicle, so people can call Data::Chronicle::Writer->VERSION and get sensible results? Normally I rely on Dist::Zilla to tag everything with a consistent version, since there's only 3 modules maybe adding $VERSION manually isn't so bad, easy to get out of sync though.